### ISSUE TITLE
optional deps should be dev deps?

### ISSUE THREAD

**Issue by kellyselden** [mike-north/ember-lodash#158]
We were getting strange issues:

```
yarn install v1.8.0
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
warning cs-core-api > ember-lodash > semantic-release > @semantic-release/npm > npm > npm-registry-fetch@1.1.1: this version has a breaking change. use 1.1.0 or upgrade to latest
error Couldn't find package "travis-deploy-once@^5.0.9" required by "@mike-north/js-lib-semantic-release-config@1.0.0" on the "npm" registry.
```

Which lead me to question, why are the CI/deployment scripts part of the dependencies. Shouldn't they be moved to devDependencies?

**Comment by kellyselden** [comment]
our yarn.lock exploded when updating to the newest version too.

**Comment by john-griffin** [comment]
Just noticed this too when checking the `yarn.lock` diff.

**Comment by mike-north** [comment]
fix in the pipe for this. New release going out tonight

**Comment by mike-north** [comment]
released as `4.19.2`
